### ISSUE TITLE
Need to see output during scan

### ISSUE THREAD

**Issue by Ecofoot** [cadeyrn/bookmarks-organizer#227]
=====================
Bookmarks Organizer needs:
 1) a "pause" button
2) List should be output to screen so one may see what it is finding
3) Results should be output-ed to a csv etc. file for further exploration
4) Ought to be able to stop scan at any point and see output

**Comment by Illegal-Services** [comment]
I agree these options could useful, the output to .CSV file is kindda unnecessary tho because we can literally do a CTRL+A keyboard shortcut in order to copy all text, from either or both "Errors" and "Warnings".

**Comment by PatrickJRed** [comment]
+1 for 1. "pause" button and 4. scanstop output
